carapace-server/src/main/java/org/carapaceproxy/server/RequestHandler.java

```diff
@@ -821,7 +821,8 @@ public void badErrorOnRemote(Throwable cause) {
     public static final String PROPERTY_CONTENT_TYPE = "request.content-type";
     public static final String PROPERTY_HEADERS = "request.headers.";
     private static final int HEADERS_SUBSTRING_INDEX = PROPERTY_HEADERS.length();
-    public static final String PROPERTY_LISTENER_ADDRESS = "listener.address";
+    public static final String PROPERTY_LISTENER_HOST_PORT = "listener.address";
+    public static final String PROPERTY_LISTENER_IPADDRESS = "request.serverip";
```

Comment:
The value of the properties has to be the same of the variable imho:

- PROPERTY_LISTENER_HOST_PORT = "listener.hostport"
- PROPERTY_LISTENER_IPADDRESS = "listener.idaddress"